pkg/component/kubernetes/controllermanager/controllermanager_test.go

```diff
@@ -747,18 +651,17 @@ namespace: kube-system
 
 				Expect(kubeControllerManager.Deploy(ctx)).To(Succeed())
 
-				verifyDeployment(config, hvpaConfig, controllerWorkers, true)
+				verifyDeployment(config, isScaleDownDisabled, controllerWorkers, true)
 			},
 
-			Entry("w/o config", emptyConfig, hvpaConfigDisabled, controllerWorkers),
-			Entry("with HVPA", emptyConfig, hvpaConfigEnabled, controllerWorkers),
-			Entry("with HVPA and custom scale-down update mode", emptyConfig, hvpaConfigEnabledScaleDownOff, controllerWorkers),
-			Entry("with non-default autoscaler config", configWithAutoscalerConfig, hvpaConfigDisabled, controllerWorkers),
-			Entry("with feature flags", configWithFeatureFlags, hvpaConfigDisabled, controllerWorkers),
-			Entry("with NodeCIDRMaskSize", configWithNodeCIDRMaskSize, hvpaConfigDisabled, controllerWorkers),
-			Entry("with PodEvictionTimeout", configWithPodEvictionTimeout, hvpaConfigDisabled, controllerWorkers),
-			Entry("with NodeMonitorGradePeriod", configWithNodeMonitorGracePeriod, hvpaConfigDisabled, controllerWorkers),
-			Entry("with disabled controllers", configWithNodeMonitorGracePeriod, hvpaConfigDisabled, controllerWorkersWithDisabledControllers),
+			Entry("w/o config", emptyConfig, false, controllerWorkers),
+			Entry("with HVPA and custom scale-down update mode", emptyConfig, true, controllerWorkers),
```

Comment:
See above: Entry description still hinting to HVPA, but this is about `scaleDownDisabled`

Reply:
Done.